ocaml/xapi/xapi_bond.ml

```diff
@@ -277,8 +277,8 @@ let create ~__context ~network ~members ~mAC ~mode ~properties =
 
   (* Validate MAC parameter; note an empty string is OK here, since that means 'inherit MAC from
      	 * primary slave PIF' (see below) *)
-  let didClientSpecifyMAC = mAC <> "" in
-  if didClientSpecifyMAC && (not (Helpers.is_valid_MAC mAC)) then
+  let did_client_specify_MAC = mAC <> "" in
```

Comment:
It's probably a good idea to use "user" rather than "client". The second one may have more than one meaning depending on the context :)